src/richie/apps/courses/models.py

```diff
@@ -112,6 +112,11 @@ class Course(BasePageExtension):
         related_name="main_courses",
         limit_choices_to={"extended_object__publisher_is_draft": True},
     )
+    course_run = models.ManyToManyField(
+        "CourseRun",
+        related_name="course_run",
```

Comment:
The related_name is the name of the relation from Course to CourseRun objects as seen from CourseRun object. So as a logical convention, it should be called `courses`, then when handling a CourseRun, you will access its Courses this way: course_run.courses (note that other relations like `organizations` use this convention)
Also as a M2M, this name should be plural: course_runs (again see `organisations` 😉  )

Reply:
I'll make the required changes. Thanks :)